components/gfx/font_cache_thread.rs

```diff
@@ -216,7 +216,8 @@ impl FontCache {
                     url: url.clone(),
                     type_: RequestType::Font,
                     destination: Destination::Font,
-                    origin: url.clone(),
+                    // TODO: Add a proper origin - Can't import GlobalScope from gfx
```

Comment:
I can't `use script::dom::globalscope::GlobalScope;` from here, because `dom` is private. Also I can't build the origin from anywhere else (I think?).